Installation issue torchzlib luarocks make rocks/torchzlib-1.0-1.rockspec

After building the module, I am getting the following error. How can I resolve it and install?

[ 94%] Building C object CMakeFiles/torchzlib.dir/torchzlib.c.o
In file included from /home/avinash/ProgramFiles/torchzlib/generic/torchzlib.c:19:0,
                 from generic/torchzlib.c:2,
                 from /home/avinash/ProgramFiles/torchzlib/torchzlib.c:20:
/home/avinash/ProgramFiles/torchzlib/zlib/zlib.h:34:19: fatal error: zconf.h: No such file or directory
compilation terminated.
make[2]: *** [CMakeFiles/torchzlib.dir/torchzlib.c.o] Error 1
make[1]: *** [CMakeFiles/torchzlib.dir/all] Error 2
make: *** [all] Error 2

Error: Build error: Failed building

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,6 +5,8 @@ FIND_PACKAGE(Torch REQUIRED)
 
 ADD_SUBDIRECTORY(zlib)
 
+INCLUDE_DIRECTORIES(${CMAKE_BINARY_DIR}/zlib)
+
 SET(src torchzlib.c)
 
 FILE(GLOB luasrc *.lua)

